### PR TITLE
Action Cable for reservation messages

### PR DESCRIPTION
**Pull request by ski-climb**
@tmikeschu styling!  Halp!

### CODE REVIEW

File: spec/rails_helper.rb [hunk 1]
```diff
@@ -14,6 +14,7 @@
   config.cassette_library_dir = Rails.root.join("spec","fixtures","vcr_cassettes")
   config.ignore_hosts 'maps.googleapis.com'
   config.hook_into :webmock
+  config.ignore_localhost = true
```
Comment (tmikeschu) [inline spec/rails_helper.rb:17]: Curious to hear about this...

File: config/application.rb [hunk 2]
```diff
@@ -21,6 +21,7 @@ class Application < Rails::Application
     # Use the responders controller from the responders gem
     config.app_generators.scaffold_controller :responders_controller
 
+    config.time_zone = "Mountain Time (US & Canada)"
```
Comment (tmikeschu) [inline config/application.rb:25]: Interesting!

File: app/controllers/messages_controller.rb [hunk 3]
```diff
@@ -4,7 +4,12 @@ class MessagesController < ApplicationController
   def create
     reservation = Reservation.find(params[:reservation_id])
     @message = reservation.messages.new(content: params[:message][:content], author: current_user)
-    flash[:danger] = "Message content can't be blank" unless @message.save
-    redirect_to user_reservation_path(current_user, reservation)
+    if @message.save
```
Comment (tmikeschu) [inline app/controllers/messages_controller.rb:7]: This looks like the moneymaker

**Review by tmikeschu — APPROVED** [review]
Nicely done, sir. Can't wait to hear more about it!

**Comment by njgheorghita** [comment]
Great work @ski-climb, actioncable looks pretty neat, looking forward to hearing more about how it works

**Review by njgheorghita — APPROVED** [review]

**Comment by Sh1pley** [comment]
nanananananana Git Merge!

**Pull request merged**